roles/keycloak/tasks/main.yml

```diff
@@ -20,10 +20,22 @@
     namespace: openstack
   register: _pxc_service
 
-- name: Install Kubernetes python package
+- name: Install MySQL python package
   ansible.builtin.pip:
     name: PyMySQL
 
+- name: Check MySQL ready
+  community.mysql.mysql_info:
+    login_host: "{{ _pxc_service.resources[0].spec.clusterIP }}"
+    login_user: root
+    login_password: "{{ openstack_helm_endpoints.oslo_db.auth.admin.password }}"
+    filter:
+      - version
+  register: mysql_ready
+  until: mysql_ready is not failed
+  retries: 10
+  delay: 5
+
```

Comment:
@ricolin i suggest making PRs separately for image-manifest fix and keycloak fix.

Reply:
I'm fine either way, but now both stuck at the same time though. It requires both to unblock IMO. Do we allow with force merging?